kouncil-frontend/apps/kouncil/src/app/login/login.component.ts

```diff
@@ -0,0 +1,52 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {AuthService} from './auth.service';
+import {Router} from '@angular/router';
+import {configProviderFactory} from '../app.module';
+import {ServersService} from '@app/common-servers';
+import {Backend} from '@app/common-model';
+import {environment} from '../../environments/environment';
+
+@Component({
+  selector: 'app-login',
```

Comment:
Can you generate nx lib, move this component and similiar functionality over there?
You can call it common-login or feature-login etc
We have nx over here, please you libs :)